resources/views/pages/user-shifts.html

```diff
@@ -47,7 +47,10 @@ <h1>%title% %add_link%</h1>
                     <button type="button" class="btn btn-secondary set-time" data-hours="8">%set_next_8h%</button>
                 </div>
                 <div class="btn-group mb-1" role="group">
-                    %buttons%
+                    %random%
```

Comment:
could you please make a config option for that?

Reply:
I don't think that it would be useful for anyone to disable it?

Reply:
I don't think that it would be useful for anyone to disable it?